### PR TITLE
improve cdk template

### PR DESCRIPTION
**Pull request by lukeymyuan**
Fix / improve a few things for CDK template

- `version` in unit test, in pipeline `version` is always going to be different
- `dd_tags` in unit test, in pipeline we do not get `dd_tags` value
- add missing start command in `package.json`
- add required smoke test configuration step in readme

### CODE REVIEW

File: template/lambda-sqs-worker-cdk/README.md [hunk 1]
```diff
@@ -14,9 +14,10 @@ Next steps:
 3. [ ] Add the repository to BuildAgency;
        see our internal [Buildkite Docs] for more information.
 4. [ ] Add Datadog extension, deployment bucket configuration and data classification tags to [infra/config.ts](infra/config.ts).
-5. [ ] Push local commits to the upstream GitHub branch.
-6. [ ] Configure [GitHub repository settings].
-7. [ ] Delete this checklist 😌.
+5. [ ] Configure smoke test `MessageAttributes` in [src/services/pipelineEventSender.ts](src/services/pipelineEventSender.ts) and `sourceSnsTopicArn` in [infra/config.ts](infra/config.ts).
```
Comment (samchungy) [inline template/lambda-sqs-worker-cdk/README.md:17]: >Configure smoke test `MessageAttributes` in [src/services/pipelineEventSender.ts](src/services/pipelineEventSender.ts)

Is this needed?

File: .changeset/quick-dragons-punch.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'skuba': patch
+---
+
+test/lambda-sqs-worker-cdk: fix unit test, add start command, and improve readme
```
Comment (samchungy) [inline .changeset/quick-dragons-punch.md:5]:
```suggestion
lambda-sqs-worker-cdk: Fix failing unit tests and add `start` command
```

File: template/lambda-sqs-worker-cdk/README.md (hunk 1 above)
Comment (lukeymyuan) [reply template/lambda-sqs-worker-cdk/README.md]: updated

additional permission to allow lambda to publish sns may be needed, I added in readme

Comment (samchungy) [reply template/lambda-sqs-worker-cdk/README.md]: Do you think we should configure the permission by default?

**Review by samchungy — APPROVED** [review]

File: .changeset/quick-dragons-punch.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'skuba': patch
+---
+
+test/lambda-sqs-worker-cdk: Fix failing unit test and add `start` command
```
Comment (samchungy) [inline .changeset/quick-dragons-punch.md:5]:
```suggestion
template/lambda-sqs-worker-cdk: Fix failing unit test and add `start` command
```

**Pull request merged**